jerry-ext/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ target_link_libraries(${JERRY_EXT_NAME} jerry-core)
 
 set(JERRY_EXT_PKGCONFIG_LIBS)
 
-if(USING_MSVC AND JERRY_DEBUGGER)
+if(WIN32 AND JERRY_DEBUGGER)
```

Comment:
tests/unit-doc/CMakeLists.txt also has a more general test for Windows platform (not only testing for the compiler) as `if("${PLATFORM}" STREQUAL "WINDOWS")`. Would that work here, too? It would be preferred to use the same tests throught the build system.